docs/k8s/user-guide.mdx

```diff
@@ -465,6 +465,90 @@ spec:
                   number: <service-port>
 ```
 
+## Traffic Policy
+
+Desigining and applying just traffic policies to inbound and outbound traffic is a simplified best practice for handling edge management.
+
+### Design
+
+Defining a traffic policy resource with rules composed of expressions and actions allows you to influence and control traffic via the policy engine.
```

Comment:
This language might be a little clearer.
```suggestion
`NgrokTrafficPolicy` objects can be defined with rules composed of expressions and actions that validate and filter traffic via the policy engine.
```

Reply:
This was tricky, I got caught up in the `influx and control traffic` phrasing for traffic policies in `docs/http/index.mdx` and the three other `docs/{http,tcp,tls}/traffic-policy/index.mdx`, but this nails it.